src/docs-assembler/Navigation/GlobalNavigationPathProvider.cs

```diff
@@ -98,7 +98,11 @@ public GlobalNavigationPathProvider(GlobalNavigationFile navigationFile, Assembl
 		}
 
 		var originalPath = Path.Combine(match.Host, match.AbsolutePath.Trim('/')).TrimStart('/');
-		var newRelativePath = relativePath.AsSpan().TrimStart(originalPath).TrimStart('/').ToString();
+		var relativePathSpan = relativePath.AsSpan();
```

Comment:
LGTM! We have a few misuses of `TrimStart()` on `Span<T>` 😢 

Maybe as a follow up PR put this in a new extension method `TrimStartExact()` and update other places we misused `TrimStart()`.